src/app/inventory/ItemIcon.m.scss

```diff
@@ -72,12 +71,20 @@ $commonBg: #366f42;
   background-position: center;
   background-size: calc((var(--item-size) - #{2 * $item-border-width}) * (96 / 90))
     calc((var(--item-size) - #{2 * $item-border-width}) * (96 / 90));
-
+}
+.legendaryMasterwork {
   background-image: url('../../images/masterwork.png');
 }
 .exoticMasterwork {
   background-image: url('../../images/exotic-masterwork.png');
 }
+.deepsightBorder {
+  background-image: url('../../images/deepsight.png');
```

Comment:
Not 100% sold on the thicker border overlay - but I'd have to use it to be sure and I don't have any of these weapons yet.

Reply:
Yeah, I'm not totally happy with it either but was seeing complaints that shaped weapons (which currently have the 1px red border) were hard to identify. With the red border moving to Deepsight, I figured I would try get ahead of this and make the border a bit thicker.

If it helps, here's how it looks alongside some other items:
![image](https://user-images.githubusercontent.com/17512262/156702569-336d4806-8817-4de4-95e5-9462e0cac95e.png)